views/default/elgg.css.php

```diff
@@ -51,6 +52,7 @@
 echo elgg_view('elements/layout.css', $vars);
 echo elgg_view('elements/misc.css', $vars);
 echo elgg_view('elements/misc/spinner.css', $vars);
+echo elgg_view('elements/fa.css', $vars);
```

Comment:
Personally I don't think the elements/* prefix is necessary anymore -- it was just left in for BC with 1.x themes. I'd like to move away from it (and from further bloating elgg.css) as much as possible.